.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -30,6 +30,8 @@ body:
         - "3.9"
         - "3.10"
         - "3.11"
+        - "3.12"
+        - "3.13alpha/beta"
```

Comment:
I don't think we should test on 3.13 until it's stable release

Reply:
It's not the test list, it's the bug report template dropdown menu, making it possible for submitting a bug report when running any currently available Python version:

![image](https://github.com/netbox-community/pynetbox/assets/25136274/c2129b1c-2d2f-49b8-81be-54c097a6d55f)

Reply:
We will add it when we are running tests for it. I don't want to give an illusion to the end user that this version is also supported